sbt-cross/src/main/scala/sbtcross/SbtCrossPlugin.scala

```diff
@@ -0,0 +1,31 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object SbtCrossPlugin extends AutoPlugin {
```

Comment:
Conventionally, an sbt plugin's `AutoPlugin` is not prefixed with `Sbt`. This should be named `CrossPlugin`.